docs/DesignDocs/AttestationApiRequirements.md

```diff
@@ -0,0 +1,291 @@
+Proposed Attestation API Requirements
```

Comment:
Overall, my understanding is that the OE SDK attestation API is focused on evidence generation and verification, at a level lower than the RATS model and does not have functions for the Relying Party role.

My thinking is that a RATS level API can be built on top of the OE SDK attestation API.

Reply:
My thinking is that a RATS level API should (at least eventually) be an OE SDK attestation API, not two different layers.